web/package.json

```diff
@@ -35,6 +35,7 @@
     "@picocss/pico": "^2.0.6",
     "@turf/along": "^7.1",
     "@turf/distance": "^7.2.0",
+    "flag-icons": "^7.3.2",
```

Comment:
28KB minified CSS, not sure if magic tree-shaking happens or not, but it's not big